doc/setup/install.rst

```diff
@@ -265,6 +266,33 @@ Source installation requires manually installing dependencies:
   - on OS X, `postgres.app <http://postgresapp.com>`_ is the simplest way to
     get started, then create a postgres user as on Linux
 
+  - on FreeBSD, install from packages, then configure it to autostart and
+    initialize it, add an user named like your login (used 'odoo' as an 
+    example) and finally set its default enconding to UTF-8:
+
+    .. code-block: console
+
+        # pkg install postgresql94-server postgresql94-client postgresql94-contrib
```

Comment:
Could you organize the following lines in a similar way to what is done with the windows section.

I have in mind something like:
```
- Install the packages:
# pkg install postgresql94-server postgresql94-client postgresql94-contrib
- Enable autostart:
echo 'postgresql_enable="YES"' >> /etc/rc.conf
echo 'postgresql="-D /usr/local/pgsql/data"' >> /etc/rc.conf
- Initialize database:
# su pgsql
$ initdb /usr/local/pgsql/data
...
```

I think it will more clear than having 20 commands one after the other. You can also get rid of the "exit", "\q", to keep only the necessary commands.